components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/LifeCycleUtils.java

```diff
@@ -102,6 +100,19 @@ public static void changeLifecycle(String user, APIProvider apiProvider, String
         }
     }
 
+    private static void UpdateLifeCycleState(APIProvider apiProvider, String orgId, ApiTypeWrapper apiTypeWrapper,
```

Comment:
Let's change this method name to `updateLifeCycleState`

Reply:
done by https://github.com/wso2/carbon-apimgt/pull/12168/commits/27e7b0edcdab808f7a4efe9b2a366368cfca0652